HTTPShortcuts/app/src/main/res/values-es/strings.xml

```diff
@@ -257,6 +257,8 @@ Haga clic en el botón + para añadir una variable.</string>
     <string name="import_in_progress">Importando…</string>
     <!-- Label, for option to pick the timeout of a shortcut. clicking it opens a picker dialog -->
     <string name="label_timeout">Se acabó el tiempo</string>
+    <!-- Label, for dropdown picker, allowing to select the Internet Protocol version used for hostname resolution by a shortcut -->
+    <string name="label_ip_version">Versión del protocolo</string>
```

Comment:
I only speak English and Spanish, so I can only contribute with text in these languages